src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const items = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+function convertNumber(str) {
+  const newStr = str.slice(1);
+
+  return +newStr.split(',').join('');
+}
+
+function sortList(arr) {
+  const sortArr = Array.from(arr);
+
+  sortArr.sort((a, b) => {
+    const firstSalary = convertNumber(a.getAttribute('data-salary'));
+    const secondSalary = convertNumber(b.getAttribute('data-salary'));
+
+    return secondSalary - firstSalary;
+  });
+
+  for (const item of sortArr) {
+    list.appendChild(item);
+  }
+
+  return sortArr;
+}
+
+function getEmployees() {
+  const listOfEmployees = document.getElementsByTagName('ul')[0];
+  const listItems = [...listOfEmployees.children];
```

Comment:
You can simply write
```suggestion
const listItems = [...list.children];
```